### PR TITLE
fix(assistant): add disable parameter to create_dialog_node

### PR DESCRIPTION
**Pull request by mamoonraja**

### CODE REVIEW

File: lib/ibm_watson/assistant_v1.rb [hunk 1]
```diff
@@ -1897,10 +1897,11 @@ def list_dialog_nodes(workspace_id:, page_limit: nil, include_count: nil, sort:
...
+    # @param disable [Bool] Whether to consider the dialog node during runtime evaluation. Set to `true` to ignore the dialog node.
     # @param user_label [String] A label that can be displayed externally to describe the purpose of the node to
     #   users. This string must be no longer than 512 characters.
     # @return [DetailedResponse] A `DetailedResponse` object representing the response.
-    def create_dialog_node(workspace_id:, dialog_node:, description: nil, conditions: nil, parent: nil, previous_sibling: nil, output: nil, context: nil, metadata: nil, next_step: nil, actions: nil, title: nil, node_type: nil, event_name: nil, variable: nil, digress_in: nil, digress_out: nil, digress_out_slots: nil, user_label: nil)
+    def create_dialog_node(workspace_id:, dialog_node:, description: nil, conditions: nil, parent: nil, previous_sibling: nil, output: nil, context: nil, metadata: nil, next_step: nil, actions: nil, title: nil, node_type: nil, event_name: nil, variable: nil, digress_in: nil, digress_out: nil, digress_out_slots: nil, disable: nil, user_label: nil)
```
Comment (mediumTaj) [inline lib/ibm_watson/assistant_v1.rb]: Mamoon, I just talked to @rrberry - `disabled` should only be in the dialog response, not in `create_dialog_node`. can you remove this?

File: lib/ibm_watson/assistant_v1.rb [hunk 2]
```diff
@@ -1930,6 +1931,7 @@ def create_dialog_node(workspace_id:, dialog_node:, description: nil, conditions
         "digress_in" => digress_in,
         "digress_out" => digress_out,
         "digress_out_slots" => digress_out_slots,
+        "disable" => disable,
```
Comment (mediumTaj) [inline lib/ibm_watson/assistant_v1.rb]: This should be removed as well.

File: lib/ibm_watson/assistant_v1.rb (hunk 1 above)
Comment (mamoonraja) [reply lib/ibm_watson/assistant_v1.rb]: ok, we can just close the PR then.

**Comment by mediumTaj** [comment]
Closing this PR since `disabled` should only be in the response.

**Pull request closed without merging**